docs/browse-list.md

```diff
@@ -1,35 +1,43 @@
 # Browse Lists
-## URLs
-https://catalog.princeton.edu/browse/subjects?search_field=browse_subject&q=. 
-https://catalog.princeton.edu/browse/names?search_field=browse_name&q=
-https://catalog.princeton.edu/browse/name_titles?search_field=name_title&q=
-https://catalog.princeton.edu/browse/call_numbers?search_field=browse_cn&q=
+## URLs for the different Browse lists
+* [Subject Browse](https://catalog.princeton.edu/browse/subjects?search_field=browse_subject&q=.)
+* [Name Browse](https://catalog.princeton.edu/browse/names?search_field=browse_name&q=)
+* [Title Browse](https://catalog.princeton.edu/browse/name_titles?search_field=name_title&q=)
+* [Call Number Browse](https://catalog.princeton.edu/browse/call_numbers?search_field=browse_cn&q=)
 
 Note: This documentation was migrated from https://docs.google.com/document/d/1bHvgfgyUmDXV7ROqEZFaRxJFEYhSSUbeQoQT883GMWg/edit#heading=h.oz6ax25g11u and needs to be updated
 
 Notice that the rake tasks indicated in this document are in **Orangelight**, not in bibdata.
 
 ## How to know which machine to use
 
-- Check [schedule.rb in Orangelight](https://github.com/pulibrary/orangelight/blob/main/config/schedule.rb#L27) to find which machine a rake task is running on.
+- Check [schedule.rb in Orangelight](https://github.com/pulibrary/orangelight/blob/main/config/schedule.rb#L27) to find which logical machine a rake task is running on.  Then see which host that is in [production.rb](https://github.com/pulibrary/orangelight/blob/main/config/deploy/production.rb#L19)
 
```

Comment:
maybe add the line range were the indexers are https://github.com/pulibrary/orangelight/blob/main/config/deploy/production.rb#L19-L21